examples/pagination.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/xanzy/go-gitlab"
+)
+
+func pagination() {
+	git := gitlab.NewClient(nil, "yourtokengoeshere")
+	git.SetBaseURL("https://gitlab.com/api/v4")
```

Comment:
This is only needed if you want to use a custom URL. So I would not put it here using the default URL 👍

Reply:
Fixed in the new changes.